wrappers/services_gen_test.go

```diff
@@ -68,6 +68,28 @@ Type=%s
 WantedBy=multi-user.target
 `
 
+const expectedUserServiceFmt = `[Unit]
+# Auto-generated, DO NOT EDIT
+Description=Service for snap application snap.app
+After=
```

Comment:
I've adjusted the code to construct the full "After" list in Go code rather than trying to do it in the template. That made it fairly easy to omit the line when empty.